README.md

```diff
@@ -121,32 +125,32 @@ output of a github-script step. For some workflows, string encoding is preferred
 By default, requests made with the `github` instance will not be retried. You can configure this with the `retries` option:
 
 ```yaml
-- uses: actions/github-script@v7
+- uses: actions/github-script@v8
   id: my-script
   with:
     result-encoding: string
     retries: 3
     script: |
-      github.rest.issues.get({
+      octokit.rest.issues.get({
```

Comment:
Reverting the README changes from #557 - with #508 we should steer users towards using `octokit` as that will be more documentation and AI friendly 😄